### ISSUE TITLE
[Feature] GetModule/ReadModule API

### ISSUE THREAD

**Issue by Swivelgames** [sysgears/webpack-virtual-modules#157]
- [x] I'd be willing to implement this feature

## Describe the user story

As a Webpack Consumer, I would like to be able to query the VirtualModulesPlugin, so that I can inspect or modify the source for my Virtual Module at a later time if necessary.

## Describe the solution you'd like

```javascript
const virtualModules = new VirtualModulesPlugin();

var compiler = webpack({
  // ...
  plugins: [
    virtualModules
  ]
});

virtualModules.writeModule(
  'node_modules/module-foo.js',
  'module.exports = { foo: "foo" };'
);

const myMod = virtualModules.readModule('node_modules/module-foo.js'); // module.exports = { foo: "foo" };

// If the goal is to update it, you can just call `writeModule` again:
virtualModules.writeModule(
  'node_modules/module-foo.js',
  myMod.replace(/"foo"/, '"bar"')
);

```

## Describe the drawbacks of your solution

Pure addition. Should not have any impact to existing functionality.

## Describe alternatives you've considered

This disgusting, fragile, private-api-harassing behemoth:
```javascript
const virt = new VirtualModulesPlugin();

virt.writeModule(
    'node_modules/module-foo.js',
    'module.exports = { foo: "foo" };'
);

// ...

const { _staticModules: mods } = virt;

const key = Object.keys(mods).find(
    f => f.indexOf('node_modules/module-foo.js') > -1
);

mods[key] = mods[key].replace(/"foo"/, '"bar"');
```

**Comment by larixer** [comment]
@Swivelgames No objections to this addition. Are you willing to submit a PR?

**Comment by larixer** [comment]
Closed via #170
